config/import-control.xml

```diff
@@ -7,7 +7,18 @@
     <allow pkg="java.io"/>
     <allow pkg="java.util"/>
     <allow pkg="com.github.checkstyle.regression.data"/>
+    <allow pkg="com.github.checkstyle.regression.util"/>
+    <allow pkg="org.immutables"/>
     <subpackage name="git">
         <allow pkg="org.eclipse.jgit"/>
     </subpackage>
+    <subpackage name="module">
+        <allow pkg="com.google.gson"/>
+        <!-- Keep this until we don't need hardcode ModuleInfo map. -->
+        <allow class="java.nio.charset.Charset"/>
+    </subpackage>
+    <subpackage name="util">
+        <allow pkg="com.google.gson"/>
```

Comment:
I am fine with `util` package, but it seems to me we are slightly breaking component design by having a component specific import outside it's package.
`gson` will never be used by another component besides `module`, right?
Should this `util` package be located inside `module`?

Looking at the `ModuleUtil`, it seems like we are breaking apart the one `module` class in the wrong way.
Anything releated to the file reading should be in a class by itself, either in a `util` package or not.
Either everything should be in the main class for the component, or split by main and change parsing. This sound good?